src/test/java/plugins/JobDslPluginTest.java

```diff
@@ -817,6 +820,365 @@ public void should_alert_if_generated_job_changed() {
         assertThat(driver, hasContent(alert));
     }
 
+    /**
+     * Tests whether a new job is created by JobDsl when the seed job is build.
+     */
+    @Test
+    public void should_create_new_job() {
+
+        // Arrange
+        String jobName = "MyJob";
+        String jobDslScript = String.format("job('%s')", jobName);
+        Job seed = createSeedJobWithJobDsl(jobDslScript);
+
+        // Act
+        seed.startBuild().waitUntilFinished();
+
+        // Assert
+        Job job = jenkins.jobs.get(Job.class, jobName);
+        assertThat(job, pageObjectExists());
+    }
+
+    /**
+     * Tests whether a job description is registered to a new job build by JobDsl
+     * when the seed job is build.
+     */
+    @Test
+    public void should_create_job_with_description() {
+
+        // Arrange
+        String jobName = "MyJob";
+        String jobDescription = "My sample despription";
+        String jobDslScript = String.format("job('%s') { description('%s') }", jobName, jobDescription);
+        Job seed = createSeedJobWithJobDsl(jobDslScript);
+
+        // Act
+        seed.startBuild().waitUntilFinished();
+
+        // Assert
+        Job job = jenkins.jobs.get(Job.class, jobName);
+        job.open();
+        WebElement descriptionElement = job.find(by.xpath("//*[@id=\"description\"]/div[1]"));
```

Comment:
move this code to TopLevelItem.getDescription so you can write
`assertThat(job.getDescription(), containsString("bla")`